src/main/java/com/mg105/controllers/TutorialTextController.java

```diff
@@ -0,0 +1,29 @@
+package com.mg105.controllers;
+
+import com.mg105.entities.GiveTutorial;
+import com.mg105.use_cases.PlayerGetsTutorial;
+import com.mg105.utils.TutorialTexts;
+
+public class TutorialTextController {
+
+    public PlayerGetsTutorial tutorial = new PlayerGetsTutorial(TutorialTexts.PHASES, 0, new GiveTutorial(false, false, false));
```

Comment:
Technically this attribute should be private (because they prefer making things private and having getters and setters instead). If you really need to view this attribute write a get method